coveo-functools/README.md

```diff
@@ -140,28 +336,30 @@ payload: Dict[str, Any] = {"inner": {}}
 _ = fn(**payload)
 ```
 
-### unflex
+# unflex
 
-Unflex is one of the utilities used by the @flex decorator.
+Unflex is one of the utilities used by `flex.deserializer`.
 
-It can remap a dictionary to fit the keyword arguments (casing/etc) given by a callable:
+It is responsible to adjust the keyword arguments of a dictionary, so they match the argument names of a target function.

```

Comment:
```suggestion
It is responsible for adjusting the keyword arguments of a dictionary, so that they match the argument names of a target function.

```